chunker/src/main/java/edu/illinois/cs/cogcomp/chunker/main/ChunkerTrain.java

```diff
@@ -99,8 +180,17 @@ public void writeModelsToDisk(String dir, String modelName){
         System.out.println("Done training, models are in " + dir+File.separator+modelName+".lc (.lex)");
     }
     public static void main(String[] args) {
-        ChunkerTrain trainer = new ChunkerTrain();
-        trainer.trainModels();
-        trainer.writeModelsToDisk();
+        if(args.length!=4&&args.length!=5){
+            System.out.println("Usage1: ...ChunkerTrain traindata modeldir modelname round");
+            System.out.println("Usage2: ...ChunkerTrain traindata modeldir modelname max_round dev_ratio");
```

Comment:
This is a non-deterministic problem with the CI build system: it sometimes fails to download all the NER resource jars and this is the message that results.